portal-ui/tests/permissions/bucketDeleteAllVersions.ts

```diff
@@ -0,0 +1,48 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import * as roles from "../utils/roles";
+import * as elements from "../utils/elements";
+import * as functions from "../utils/functions";
+import { bucketsElement } from "../utils/elements-menu";
+import { testBucketBrowseButtonFor } from "../utils/functions";
+import { Selector } from "testcafe";
+import * as constants from "../utils/constants";
+
+fixture("For user with Bucket Delete permissions").page("http://localhost:9090");
```

Comment:
Hi @adfost, let's change this text as we're running the test for a user that has Read and Write permissions (i.e., we're using the `bucketReadWrite` role):

```suggestion
fixture("For user with Bucket Read & Write permissions").page("http://localhost:9090");
```